src/main/java/com/liveramp/captain/waypoint/WaypointSubmitterImpl.java

```diff
@@ -23,20 +23,20 @@
   }
 
   @Override
-  public void submitServiceRequest(long jobId, RequestContext requestOptions) throws Exception {
+  public void submitServiceRequest(long id, RequestContext requestOptions) throws Exception {
```

Comment:
does it make sense to just rename this (and any other associated functions that deal with `serviceRequest`) to `requestId` then?

Reply:
or are you suggesting renaming this method to `submit` or just saying instead of `id` we should call it `requestId`

Reply:
you comment has inspired me to think that `submitServiceRequest` should go to `submit` since "service request" is more of an internal concept for us. but i don't know if that's what you were trying to suggest. i don't think it was.

Reply:
Hm, kind've both.

If we want to keep it as `submitServiceRequest`, then I think it makes sense to change it to `requestId`. As a followup, does it actually make sense to keep it as `submitServiceRequest`? So, ya, I'd imagine we could just be called `submitWork` or something to imply submitting this number ...will do something

Reply:
+1 to rename`submitServiceRequest`.

Reply:
okay. renamed `submitServiceRequest` => `submit`. way easier than i thought.

Reply:
based on that change do we prefer `requestId` or just `id`?